code/modules/research/experimentor.dm

```diff
@@ -609,6 +600,7 @@
 	name = realName
 	cooldownMax = rand(60,300)
 	realProc = pick("teleport","explode","rapidDupe","petSpray","flash","clean","corgicannon")
+	origin_tech = pick("engineering=[rand(2,5)]","magnets=[rand(2,5)]","plasmatech=[rand(2,5)]","programming=[rand(2,5)]","powerstorage=[rand(2,5)]")

```

Comment:
Thanks to @Bawhoppen for mentioning randomized tech levels. I made strange objects have random tech levels (after using them in experimentor). This code looks a bit weird but it works perfectly. If someone has ideas how to make this look better and remove the repeating rand()s I can change this.